ext/strings.go

```diff
@@ -485,6 +490,35 @@ func join(strs []string) (string, error) {
 	return strings.Join(strs, ""), nil
 }
 
+// escape implements a string escaping function. The string will be wrapped in
+// double quotes, and all valid CEL escape sequences will be escaped to show up
+// literally if printed.
+func escape(s string) (string, error) {
+	var escapedStrBuilder strings.Builder
+	for _, c := range s {
+		switch c {
+		case '\a':
+			escapedStrBuilder.WriteString("\\a")
+		case '\b':
+			escapedStrBuilder.WriteString("\\b")
+		case '\f':
+			escapedStrBuilder.WriteString("\\f")
+		case '\n':
+			escapedStrBuilder.WriteString("\\n")
+		case '\r':
+			escapedStrBuilder.WriteString("\\r")
+		case '\t':
+			escapedStrBuilder.WriteString("\\t")
+		case '\v':
+			escapedStrBuilder.WriteString("\\v")
```

Comment:
I'm guessing you'd need to escape `\` as well.

Reply:
Right, I'll add a branch and test case for `\`.

Reply:
(marking as resolved now that branch/test case have been added)